src/index.html

```diff
@@ -16,16 +17,19 @@
       href="style.css"
     />
   </head>
+
   <body>
     <input
+      class="search-bars search-bars-1"
       type="text"
-      data-qa="keypress"
+      data-qa="big"
       placeholder="Try “Los Angeles“"
     />
 
     <input
+      class="search-bars search-bars-2"
       type="text"
-      data-qa="hover"
+      data-qa="small"
       placeholder="Try “Los Angeles“"
     />
```

Comment:
The class names 'search-bars-1' and 'search-bars-2' are not very descriptive. It's recommended to use more meaningful class names related to the content or functionality of the elements. Also, consider using BEM methodology for naming your classes.